Assets/MixedRealityToolkit/Providers/BaseController.cs

```diff
@@ -136,7 +136,7 @@ public bool SetupConfiguration(Type controllerType, InputSourceType inputSourceT
                             profileFound = true;
 
                             // If it is an exact match, assign interaction mappings.
-                            if (controllerMappings[i].Handedness == ControllerHandedness &&
+                            if ((controllerMappings[i].Handedness & ControllerHandedness) != 0 &&
```

Comment:
Be aware, that Controllers are not detected as "both", so where you might have a mapping defined as both, the Platform will still report the "hand" as either left or right.  Better to simply test if you have a "Both" definition to apply to either left or right, e.g.
> controllerMappings[i].Handedness == ControllerHandedness || controllerMappings[i].Handedness == Handedness.Both

Reply:
That's what the binary comparison is for, is it not? If controllerMappings[i].Handedness is "Both" it does not matter what hand ControllerHandedness is, it works for both.

Reply:
I would ask why you need to set it in the PoseSynchroniser?  Mainly because no SDK has a "Both" setting from the SDK, Both is an approximation for the toolkit to enable configuration that will work in either hand.

Once a controller is being visualised, it's either in the Left or Right hand, as determined from the underlying SDK

Reply:
> controllerMappings[i].Handedness is "Both" it does not matter what hand ControllerHandedness is, it works for both

This is where Any might be a better description.

Reply:
> I would ask why you need to set it in the PoseSynchroniser?

because with the Oculus Go, the controller is symmetrical (ControllerVisualization any hand), and can be held in either hand (PoseSynchronizer any hand).
It also just for convenience, because you can have a single synchronizer class and a single prefab, within which you configure what hand it is individually. Oculus's visuals, for all their platforms, comes within one prefab, so you just toggle their visibility as needed.

Reply:
> This is where Any might be a better description.

@davidkline-ms Any already exists in the enum but also includes Other as an option. This leads me to the question, why not opening this up for all handedness choices you can make?